### PR TITLE
Block Stand Elegant Crash

### PR DESCRIPTION
**Pull request by Rxann**
Part of the big PR that was broken up.

### CODE REVIEW

**Comment by lonelybud** [comment]
can someone test it using stand menu after workflow approval

**Comment by rkwapisz** [comment]
There are a bunch of unnecessary formatting changes. Please make sure you look at your commits and discard things that don't need to be reviewed.

File: src/hooks/protections/script_event_handler.cpp [hunk 1]
```diff
@@ -5,14 +5,15 @@
...
-#include "util/session.hpp"
 #include "util/script_database.hpp"
+#include "util/session.hpp"
 
 #include <network/CNetGamePlayer.hpp>
 #include <network/Network.hpp>
 #include <script/globals/GPBD_FM_3.hpp>
 #include <script/globals/GlobalPlayerBD.hpp>
 
+
```
Comment (gir489returns) [inline src/hooks/protections/script_event_handler.cpp:16]: Why are half your commits always unncessary white space changes? Major amature hour.

File: src/hooks/protections/script_event_handler.cpp [hunk 2]
```diff
@@ -476,7 +481,8 @@ namespace big
 			auto local_time = *std::localtime(&timer);
 
 			static std::ofstream log(g_file_manager.get_project_file("./script_events.log").get_path(), std::ios::app);
-			log << "[" << std::put_time(&local_time, "%m/%d/%Y %I:%M:%S") << ":" << std::setfill('0') << std::setw(3) << ms.count() << " " << std::put_time(&local_time, "%p") << "] " << output.str() << std::endl;
+			log << "[" << std::put_time(&local_time, "%m/%d/%Y %I:%M:%S") << ":" << std::setfill('0') << std::setw(3) << ms.count() << " " << std::put_time(&local_time, "%p") << "] "
+			    << output.str() << std::endl;
```
Comment (gir489returns) [inline src/hooks/protections/script_event_handler.cpp:485]: Revert

**Review by gir489returns — CHANGES_REQUESTED** [review]

**Review by Yimura — APPROVED** [review]
The random changes are a consequence of the clang-formatter no way around it unless we stop alphabetically ordering includes.

**Pull request merged**